kpconv_torch/utils/trainer.py

```diff
@@ -433,13 +410,16 @@ def cloud_segmentation_validation(self, net, val_loader, debug=False):
         # Number of classes including ignored labels
         nc_tot = val_loader.dataset.num_classes
 
+        # Number of classes predicted by the model
+        nc_model = self.num_classes
```

Comment:
I honestly don't understand the need for a variable there...

Reply:
it was to keep the comment